tests/sentence_transformers/test_sentence_transformers_model_export.py

```diff
@@ -91,11 +93,21 @@ def test_logged_data_structure(model_path, basic_model):
     ) == expected_requirements
 
     mlmodel = yaml.safe_load(model_path.joinpath("MLmodel").read_bytes())
-    assert mlmodel["flavors"]["python_function"]["loader_module"] == "mlflow.sentence_transformers"
-    assert (
-        mlmodel["flavors"]["python_function"]["data"]
-        == mlflow.sentence_transformers.SENTENCE_TRANSFORMERS_DATA_PATH
-    )
+    assert "model_size_bytes" in mlmodel
+
+    pyfunc_flavor = mlmodel["flavors"]["python_function"]
+    assert pyfunc_flavor["loader_module"] == "mlflow.sentence_transformers"
+    assert pyfunc_flavor["data"] == mlflow.sentence_transformers.SENTENCE_TRANSFORMERS_DATA_PATH
+
+    st_flavor = mlmodel["flavors"]["sentence_transformers"]
+    assert st_flavor["pipeline_model_type"] == "BertModel"
+    if Version(sentence_transformers.__version__) >= Version("2.3.0"):
+        assert st_flavor["source_model_name"] == "sentence-transformers/all-MiniLM-L6-v2"
+    else:
+        # Before Transformers 2.3.0, the library loads the Transformers model after local snapshot
```

Comment:
Is this behavior useful to the inference team? 
With all current DBR having sentence-transformers 2.2.2 installed, should we handle this by stripping the local file portion out?

Reply:
Per the feedback from Maddie, let's strip out the cache path information and parse the constructed cache name
`/some/local/cache/path/location/sentence_transformers/<hf name>_<model_name>` -> `<hf_name>/<model_name>`